src/Bundle/Controller/ResourceController.php

```diff
@@ -289,6 +292,8 @@ public function updateAction(Request $request): Response
                 return $this->redirectHandler->redirectToReferer($configuration);
             }
 
+            $this->flashHelper->addSuccessFlash($configuration, ResourceActions::UPDATE, $resource);
```

Comment:
```suggestion
            if ($configuration->isHtmlRequest()) {
                $this->flashHelper->addSuccessFlash($configuration, ResourceActions::UPDATE, $resource);
            }
```

Reply:
I don't know why I've done this only for the `createAction` XD 🦀